### PR TITLE
Added updated Scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Added Updated Scripts:
       Test_TC_DM_2_2.yaml
	Test_TC_ETHDIAG_1_1.yaml
	Test_TC_MC_2_1.yaml
	Test_TC_MC_8_1.yaml
	Test_TC_WIFIDIAG_1_1.yaml
	Test_TC_WNCV_2_1.yaml
[Execution-logs-May10.zip](https://github.com/project-chip/connectedhomeip/files/8663833/Execution-logs-May10.zip)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml [hunk 1]
```diff
@@ -28,65 +31,92 @@ tests:
...
+          value: SoftwareVersionValue
+
+    ##Event 11343 implementation not done
+    - label: "Reboot target device"
+      cluster: "SystemCommands"
+      command: "Reboot"
+      arguments:
+          values:
+              - name: "discriminator"
+                value: discriminator
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:78]:
```suggestion
```

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml [hunk 2]
```diff
@@ -28,65 +31,92 @@ tests:
...
-          value: 16
-          constraints:
-              type: uint8
+    - label: "Reboot target device"
+      cluster: "SystemCommands"
+      command: "Reboot"
+      arguments:
+          values:
+              - name: "discriminator"
+                value: discriminator
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:55]: Reboot does not take those arguments. It takes no arguments at all.
```suggestion
```

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml [hunk 3]
```diff
@@ -28,65 +31,92 @@ tests:
...
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Reboot target device"
+      cluster: "SystemCommands"
+      command: "Reboot"
+      arguments:
+          values:
+              - name: "discriminator"
+                value: discriminator
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:106]:
```suggestion
```

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:78]: i will update

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:55]: i will update

File: src/app/tests/suites/certification/Test_TC_DM_2_2.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_DM_2_2.yaml:106]: i will update

**Review by krypton36 — APPROVED** [review]

**Comment by manjunath-grl** [comment]
@bzbarsky-apple @krypton36 checks Darwin (no-ble-asan) and Darwin (no-ble-asan-clang) are failing for test DM-2.2 but works fine for Chip-tool Darwin can we skip the test DM-2.2 from CI and make it manual for now?
Issue made :https://github.com/project-chip/connectedhomeip/issues/18475

**Comment by bzbarsky-apple** [comment]
Making those manual tests for now makes sense, yes.  Or you could exclude them in just the Darwin CI, either way.

**Comment by bzbarsky-apple** [comment]
Fast-tracking test additions.

**Pull request merged**